www/static/plugins/supportedplatforms.jsx

```diff
@@ -1,83 +1,32 @@
-var Preact = require('preact'),
-    h = require('preact').h,
-    createClass = require('preact-compat').createClass,
-    PlatformButton = require('./platformbutton.jsx');
+const Preact = require('preact');
+const h = require('preact').h;
+const createClass = require('preact-compat').createClass;
+const PlatformButton = require('./platformbutton.jsx');
 
-var SupportedPlatforms = createClass({
-    render: function() {
-        var keywords = this.props.keywords;
-        var sortedPlatforms = [
-            {present:false, icon: "android", alt:"Android"},
-            {present:false, icon: "ios", alt:"iOS"},
-            {present:false, icon: "windows", alt:"Windows"},
-            {present:false, icon: "blackberry", alt:"Blackberry"},
-            {present:false, icon: "ubuntu", alt:"Ubuntu"},
-            {present:false, icon: "firefox", alt:"FirefoxOS"},
-            {present:false, icon: "webos", alt:"webOS"},
-            {present:false, icon: "fireos", alt:"FireOS"},
-            {present:false, icon: "osx", alt:"OS X"},
-            {present:false, icon: "browser", alt:"Browser"}
+const SupportedPlatforms = createClass({
+    render: function () {
+        // Collection of the npm package keywords.
+        const keywords = new Set(this.props.keywords);
```

Comment:
(plugins != docs, just in the same repo but actually at https://cordova.apache.org/plugins/)

Reply:
> IE11 doesn't support the `Set` constructor accepting arguments so this would break there. Will `Set` get transpiled here?

Good catch! I have no idea if there's transpilation. If not, we should revert my whole proposal since IE11 does not support arrow functions either.

Reply:
@raphinesse I just uploaded another alternative that doesn't use arrow functions or Set.

As for transpilation, I see a transform process that babelify, in the gulp build task but only has the react preset.

https://github.com/apache/cordova-docs/blob/0c5f1136f56a20582ab58a41757500ea79c71a3b/gulpfile.js#L406

Reply:
Out of curiosity, what is the progress of this PR? Is it considered finished or has it been delayed for `babel-preset-env` to be added? 

I believe the scope of this PR had been completed and that the `babel-preset-env` is an additional requirement for the next PR.

Reply:
@erisu To me, this PR is good to go. I did not test it though and I don't know enough about docs and plugin pages to approve or merge. @janpio might be fit to do that

Reply:
Unfortunately I don't know anything about the plugins subproject here.
But as you @raphinesse and @erisu are both ok with the current state, I think this should be safe to merge.

Reply:
@erisu Looks good to me, feel free to merge it (I'm still sorting out my Apache / Github permissions)